play.lic

```diff
@@ -0,0 +1,35 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#play
+=end
+class Play
+  def initialize
+    @no_play_scripts = get_settings.sanowret_no_use_scripts
+    @song = UserVars.song
+    main_loop
+  end
+
+  def play_song
+    fput("play #{@song}")
```

Comment:
💡 Can we use `bput` here?

Reply:
I'd prefer not since we don't validate whether the action was a success or not. If we start doing checks like scaling the instrument difficulty then we can switch it there.